tests/platform_tests/api/test_fan_drawer.py

```diff
@@ -154,3 +154,17 @@ def test_set_fan_drawers_led(self, duthost, localhost, platform_api_conn):
                             color, color_actual, i))
 
         self.assert_expectations()
+
+    def test_get_position_in_parent(self, platform_api_conn):
+        for i in range(self.num_fan_drawers):
+            position = fan_drawer.get_position_in_parent(platform_api_conn, i)
+            self.expect(position is not None, "Failed to perform get_position_in_parent for fan drawer {}".format(i))
+            self.expect(isinstance(position, int), "Position value must be an integer value for fan drawer {}".format(i))
```

Comment:
The second self.expect() call is irrelevant if the first fails, so it should be nested using if self.expect() on the first.